src/util/saveArticles.ts

```diff
@@ -402,7 +421,25 @@ async function treatVideo(mw: MediaWiki, dump: Dump, srcCache: KVS<boolean>, art
     }
 
     sourceEl.setAttribute('src', newUrl);
-    return { mediaDependencies };
+
+    /* Scrape subtitle */
+    const trackEle = videoEl.querySelectorAll('track');
+    if (trackEle) {
+        for (const track of Array.from(trackEle)) {
+            subtitles.push(await treatSubtitles(track, webUrlHost, mw, articleId));
+        }
+    }
+    return { mediaDependencies, subtitles };
+}
+
+export async function treatSubtitles(trackEle: DominoElement, webUrlHost: string, mw: MediaWiki, articleId: string): Promise<string> {
```

Comment:
I would like to have a clear differenciation in function/variable naming between `subtitle` = one track, and subtitles = multiple tracks. Here it seems you treat one track, so it should be singular form.